nvflare/app_common/workflows/cross_site_model_eval.py

```diff
@@ -125,8 +125,12 @@ def start_controller(self, fl_ctx: FLContext):
             clients = engine.get_clients()
             self._participating_clients = [c.name for c in clients]
         else:
-            assert isinstance(self._participating_clients, list), "participating_clients must be a list"
-            assert all(isinstance(x, str) for x in self._participating_clients), "participating_clients must be strings"
+            if not isinstance(self._participating_clients, list):
+                raise TypeError(
+                    "participating_clients must be a list but got {}".format(type(self._participating_clients))
+                )
+            if not all(isinstance(x, str) for x in self._participating_clients):
+                raise TypeError("participating_clients must be strings")
```

Comment:
Is it more proper to move this part in __init__?

Reply:
done